### PR TITLE
Adding subchapters in the leadership

### LINKED ISSUE #1224: Add text to "Leadership in Data Science"
### Summary

The [Leadership in Data Science](https://the-turing-way.netlify.app/collaboration/leadership.html) currently looks a bit sad with only headers.

image

If we'd add at least a bit of text, it might be a big improvement.

### What needs to be done?

- [ ] Add some intro text what this chapter will be about

### Who can help?

* Anyone who know (or can find out) what this chapter is supposed to be about.

---

### Updates

Nothing yet

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #1224 

* Adding subchapter in the Leadership chapter

### List of changes proposed in this PR (pull-request)

* Draft developed by @lauracion and @malvikasharan : https://hackmd.io/vaXifU0nR2aaOjbJOJfflw
  * Partially transferred to this PR by @malvikasharan 
* Edited in content by @malvikasharan 
* to be edited and reviewed by @bsipocz 

### What should a reviewer concentrate their feedback on?

- [ ] A landing page
- [ ] Following subchapter
  - file: collaboration/leadership
  - file: collaboration/leadership/leadership-features
  - file: collaboration/leadership/leadership-building
  - file: collaboration/leadership/leadership-structure
  - file: collaboration/leadership/leadership-diversity
  - file: collaboration/leadership/leadership-opportunities
  - file: collaboration/leadership/leadership-avoid
  - file: collaboration/leadership/leadership-stories

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Review by luucamay — APPROVED** [review]

File: book/website/collaboration/leadership.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
-## Leadership in Data Science
+(cl-leadership)=
+# Leadership in Data Science
 
-### Overview
+## Overview
+
+On the one hand, data science is **broadly and inclusively** considered as any research-driven by or conducted using data.
```
[the remark this answers is not in the document]
Comment (lauracion) [reply book/website/collaboration/leadership.md]: Thank you, Kim!

**Comment by lauracion** [comment]
Today I fixed tiny problems, so the chapter in this branch renders. 

The Figure in leadership.md is not rendering correctly and I still need to figure out why.

I need to continue fine-tuning leadership-building.md next.

**Comment by lauracion** [comment]
Next time I have a chance at working on this I need to:
* figure out what is the problem in the alt text for the figure in leadership-building.md
* find a good place to talk about [benevolent dictator for life](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)  in this chapter and how it does not align well with most of what open and healthy leadership entails
* check if the figure in leadership.md now has the right path
* continue editing leadership-structure.md
* Ne

**Comment by bsipocz** [comment]
> find a good place to talk about benevolent dictator for life in this chapter and how it does not align well with most of what open and healthy leadership entails

What about having a section with "other leadership models" with the long description of "prevalent, exclusive practices in open source /data science communities". In addition to BDFL it could include self-appointed committees with fixed membership clinging to power, basically acting as BDFL. And maybe also the "lab run" open source/data science project might also fit, e.g. when one has to be affiliated with an institute to be part of the project (the approach is all legit of course, but not as a way to lead an inclusive project).

**Comment by lauracion** [comment]
That would be great, Brigitta!
If you would like to work on adding this section, please go ahead! Happy to be one of its reviewers later on.
Also, if you have suggestions to improve the current chapter, please go ahead 😄

**Comment by bsipocz** [comment]
For review, would you prefer an inline review on the PR or comments on the draft hackmd?

I can start adding a skeleton for the section and drafting it out.

**Comment by lauracion** [comment]
Great!
Inline review on the PR is best. The hackmd draft is no longer up to date.
Thank you, Brigitta!

**Comment by malvikasharan** [comment]
@pherterich I have updated this branch here. Let me know if you face any error editing files in this chapter.

**Comment by malvikasharan** [comment]
I have created a placeholder for the notes @aleesteele took at yesterday's discussion led by Patricia, and maybe a place to also add notes from March's Fireside discussion: Emergent Roles in Research Infrastructure & Technology | Noor, Esther, Borhane, Beth, Arielle | [Video](https://www.youtube.com/watch?v=i3_GFRS-u-Q) | Etherpad: https://pad.sfconservancy.org/p/ttw-fireside-chat-mar2022

**Comment by SaraVilla** [comment]
Some thoughts/doubts/input suggestions about the 'What to avoid in leadership' subchapter:

- Personalism: not sure I understand this concept, what do you mean by it? not much to add so far...
- Elitism: Leadership and research in general benefits a lot from inclusivity and diversity. Thus, being a elitist leader won't be efficient for any project neither for the leaders themselves. The more open and different, the better
- Ignore the personal side of the team: members of the team/organisation can disclose or not as much as they want from their personal lives. But it is up to the leader to worry and keep an eye on them in order to encourage understanding, community help, help seeking and avoid burnout. If leaders don't' ask, team member most likely (and even more newer member of the team), won't ask for help. This will likely have an impact in the overall project/team/organisation. 
- Tunnel vision: leaders should be aware of the outside-organisation problems/alternatives/options... Even if the organisation is focus on one line of work, awareness about what is 'going on around' will prompt further discussions and go beyond finding quick solutions.
- Hide weaknesses: weaknesses should be something to embrace and work on, not something to hide. When accepting what a person/team is missing, there is more chances of successful solutions (if needed; sometimes a weakness is not an error or mistake that needs to be solved)

**Comment by bsipocz** [comment]
If Jim doesn't get to it, I'm happy to try to rebase and conflict resolve this in the evening (pacific time)

**Comment by malvikasharan** [comment]
That would be super helpful, Brigitta 🙏

**Comment by malvikasharan** [comment]
Created another branch with the files that are being committed here #3134

**Comment by JimMadge** [comment]
Quite a merge conflict :smile:.

Is this because of images being compressed?

@bsipocz I've never been great at this kind of merge, where you are asked about 'LOCAL' and 'REMOTE'. I suppose the answer here is to always choose 'REMOTE'?

When you do that, there are still problems with paths to images which don't exist and some `WARNING: image file not readable:`. I think it will still require some manual work to reintroduce deleted images or update paths.

**Comment by JimMadge** [comment]
@BatoolMM yes, sometimes getting your workspace correct, stage, commit and let git worry about the diff is an easier way to get the same product. I think that is legitimate. It is a shame to loose the individual commits and authors. Luckily we have other ways to recognise the contributors :+1:.

**Comment by malvikasharan** [comment]
I am giving up. will create a new branch with the files. Thank you all!

**Comment by malvikasharan** [comment]
I am seeing the same issue here: https://github.com/alan-turing-institute/the-turing-way/pull/3135

@JimMadge I think you are right that the error is because of the CI for image compression.

**Comment by malvikasharan** [comment]
Ok, created another one manually on GitHub: https://github.com/alan-turing-institute/the-turing-way/pull/3136

**Comment by bsipocz** [comment]
> Ok, created another one manually on GitHub: https://github.com/alan-turing-institute/the-turing-way/pull/3136

My week was way busier than I was expecting it, and now that the follow-up PR is merged, is this still relevant?

**Comment by malvikasharan** [comment]
No worries Brigitta. This PR is not relevant anymore. I will close this.

**Pull request closed without merging**